include/openssl/modes.h

```diff
@@ -164,6 +164,11 @@ int CRYPTO_xts128_encrypt(const XTS128_CONTEXT *ctx,
                           const unsigned char *inp, unsigned char *out,
                           size_t len, int enc);
 
+int CRYPTO_xts128gb_encrypt(const XTS128_CONTEXT *ctx,
+                          const unsigned char iv[16],
+                          const unsigned char *inp, unsigned char *out,
+                          size_t len, int enc);
+                          
```

Comment:
Please do not make this a public function.

Reply:
I added a new header file internal/modes.h to declare this function, is that OK?

Reply:
Could you put it in the existing crypto/modes.h?

Reply:
fixed in [c61f00f ](https://github.com/openssl/openssl/commit/c61f00fe773a23590d53d748da1a42d84dba8658)